src/LaraCart.php

```diff
@@ -209,6 +209,42 @@ public function addItem(CartItem $cartItem)
         return $cartItem;
     }
 
+    /**
+     * Increment the quantity of a cartItem based on the itemHash
+     *
+     * @param $itemHash
+     *
+     * @return CartItem | null
+     */
+    public function increment($itemHash)
+    {
+        $item = array_get($this->getItems(), $itemHash);
+        $item->qty++;
+
+        return $item;
+    }
+
+    /**
+     * Decrement the quantity of a cartItem based on the itemHash
+     *
+     * @param $itemHash
+     *
+     * @return CartItem | null
+     */
+    public function decrement($itemHash)
+    {
+        $item = array_get($this->getItems(), $itemHash);
+        if ($item->qty > 1) {
+            $item->qty--;
+
+            return $item;
+        }
+
+        $this->removeItem($itemHash);
+
+        return null;
```

Comment:
I'd return $this->removeItem($itemHash); but removeItem doesn't return anything itself.

Other than that can you tag this release please ? So I can composer update it ;)